golang-github-containernetworking-plugins/deb/debian/control

```diff
@@ -15,6 +15,7 @@ Vcs-Browser: https://salsa.debian.org/go-team/packages/golang-github-containerne
 Vcs-Git: https://salsa.debian.org/go-team/packages/golang-github-containernetworking-plugins.git
 XS-Go-Import-Path: github.com/containernetworking/plugins
 Testsuite: autopkgtest-pkg-go
+Provides: containernetworking-plugins
```

Comment:
Why are we using `Provides` here and not `Replaces` and `Conflicts`?  Do we even need this provides at all?

Reply:
Removed this provides line